release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3808,4 +3808,23 @@ $ oc adm release info 4.6.61 --pullspecs
 [id="ocp-4-6-61-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-62"]
+=== RHSA-2022:7306 - {product-title} 4.6.62 bug fix update
```

Comment:
```suggestion
=== RHSA-2022:7306 - {product-title} 4.6.62 bug fix and security update
```

Reply:
My fault @Wiharris this is just a RHBA or bug fix. Appreciate the catch.